attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -126,16 +111,260 @@ improvement_as_proportion_of_lifetime = 0.05 to 0.5
 value_fwi_fish = 
   fish_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime)
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime)
```

Comment:
I suggest valuing salmon in some fish/animal unit, for the correlations to work.